assets/app/js/renderer.js

```diff
@@ -204,6 +205,17 @@ $(document).ready(() => {
     return updateViews();
   }
 
+  // Get the ScanCode version and options data from the DB and populate and open the modal
+  function getScanInfo() {
+    return aboutCodeDB.sync
+      .then((db) => db.Header.findById(1)
+        .then((header) => {
+          const scancode_label = $('#scancode-label').find('#scancode-display');
+          scancode_label.text('ScanCode version: ' + header.scancode_version + '\n\nScanCode options: ' + JSON.stringify(header.scancode_options, null, 2));
+        }))
+      .then($('#myModal').modal('show'));
+  }
```

Comment:
The above getScanInfo() function works well for when there is a sqlite file loaded. However, we need to change the behavior when there is no sqlite file. 

It would be nice to:
* catch whatever exception is thrown when `getScanInfo()` is run before a valid sqlite file is loaded (see pic below)
* Display some placeholder text like: "Please import a ScanCode results file or an Aboutcode-Manager sqlite file to see the scan options" or similar

Reply:
Actually we probably don't have to catch an exception, we just need to have an if statement checking `if (header.scancode_version === null || header.scancode_options === null) {}` before crafting the string we pass to the modal.

Reply:
Good idea @majurg -- will implement.